solutions/src/main/java/ru/sberbank/school/task11/StreamsImpl.java

```diff
@@ -0,0 +1,126 @@
+package ru.sberbank.school.task11;
+
+import ru.sberbank.school.util.Solution;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+/**
```

Comment:
First sentence of Javadoc is missing an ending period.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1354560906>*